src/Chainweb/Version.hs

```diff
@@ -924,8 +924,8 @@ chainweb213Pact :: ChainwebVersion -> BlockHeight -> Bool
 chainweb213Pact Mainnet01 = (>= 2_447_358) -- 2022-02-26 00:00:18
 chainweb213Pact Testnet04 = (>= 1_977_816) -- 2022-02-25 00:00:18
 chainweb213Pact Development = (>= 95)
-chainweb213Pact (FastTimedCPM g) | g == petersonChainGraph = (>= 5)
-chainweb213Pact _ = const True
+chainweb213Pact (FastTimedCPM g) | g == petersonChainGraph = (> 25)
+chainweb213Pact _ = const False
```

Comment:
Is this needed? It means that in general tests only cover pre chainweb213 legacy behavior.

Reply:
Changed, all tests pass with it.